float32 goal_x
float32 goal_y
float32 tolerance
string turtle_name
---
bool success
